tests/test_utils.py

```diff
@@ -0,0 +1,41 @@
+from unittest.mock import Mock
+
+from lightning_utilities.core.imports import package_available
+
+from lightning_graphcore.utils import _LightningModuleWrapperBase
+
+if package_available("lightning"):
+    from lightning.pytorch.demos.boring_classes import BoringModel
+elif package_available("pytorch_lightning"):
+    from pytorch_lightning.demos.boring_classes import BoringModel
+
+
+def test_wrapper():
```

Comment:
does this need to be called in this specific sequence, or can we have it parametrized and each particular block as its own test?